apps/web/src/components/Badge/Badge.tsx

```diff
@@ -37,22 +38,17 @@ export default function Badge({ badgeRef }: Props) {
   const communityUrl = useMemo<Route>(() => {
     const contractAddress = contract?.contractAddress?.address as string;
 
-    if (contract?.chain === 'POAP') {
-      return {
-        pathname: '/community/poap/[contractAddress]',
-        query: { contractAddress },
-      };
-    } else if (contract?.chain === 'Tezos') {
-      return {
-        pathname: '/community/tez/[contractAddress]',
-        query: { contractAddress },
-      };
-    } else {
-      return {
-        pathname: '/community/[contractAddress]',
-        query: { contractAddress },
-      };
-    }
+    /* The returned chain is usually uppercase, so we have to make sure we always convert
+     them to lowercase, otherwise user's will see the inconsistent routes.
+
+     TODO: we could probablly think of a better way to handle this, so we don't have to do this manually everytime.
+    */
+    const chain = contract?.chain?.toLocaleLowerCase() as Chain;
```

Comment:
just asked chatGPT, i think maybe you could do this!
<img width="490" alt="image" src="https://github.com/gallery-so/gallery/assets/12162433/a8d83930-0db6-4547-ad91-fa5ea0ccacb4">

since this `Chain` type is auto-imported from the autogenerated `/generated/enums` file, i think you can define a new type in our existing `chains.ts` file like:

```ts
import { Chain } from '~/generated/enums';

export type LowercaseChain = Lowercase<Chain>; // import this guy
```